common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerValidator.java

```diff
@@ -229,6 +231,25 @@ private X509Certificate getSelfSignedCert(final List<X509Certificate> certs)
         return selfSignedCert;
     }
 
+    /**
+     * Returns the package that is currently active relative to the Work Account custom account type
+     * Note: either the company portal or the authenticator
+     *
+     * @param context
+     * @return String
+     */
+    public String getCurrentActiveBrokerPackageName(@NonNull final Context context) {
```

Comment:
`@Nullable` return value would be good to document with an annot / javadoc (where presumably `null` indicates no active broker)

Reply:
Oh, I see this was moved. Still good to have :)

Reply:
updated